### PR TITLE
Improve docs on how to write detectors.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: docs/How-To-Write-New-Detectors.md [hunk 1]
```diff
@@ -59,13 +59,29 @@ module Reek
...
+in those changes as well.
+
 ### Documentation
 
 * Above every `SmellDetector::sniff` method it should be documented what the expected AST is
 * Every detector should have a separate documentation page in /docs. You can
   take any arbitrary existing smell detector documentation page as template (since
   they all have the same structure already)
 * The detector should be listed under [Code Smells](docs/Code-Smells.md)
+* Depending on what your detector does it might make sense to add him to other doc pages as
```
Comment (mvz) [inline docs/How-To-Write-New-Detectors.md]: him -> it

Comment (troessner) [inline docs/How-To-Write-New-Detectors.md]: Fixed!

File: docs/How-To-Write-New-Detectors.md [hunk 2]
```diff
@@ -59,13 +59,29 @@ module Reek
...
+
 ### Documentation
 
 * Above every `SmellDetector::sniff` method it should be documented what the expected AST is
 * Every detector should have a separate documentation page in /docs. You can
   take any arbitrary existing smell detector documentation page as template (since
   they all have the same structure already)
 * The detector should be listed under [Code Smells](docs/Code-Smells.md)
+* Depending on what your detector does it might make sense to add it to other doc pages as
+  well e.g. [Code Smells](docs/Simulated-Polymorphism.md)
```
Comment (mvz) [inline docs/How-To-Write-New-Detectors.md]: Missed this first time round: Code Smells should read Simulated Polymorphism here.

Comment (troessner) [inline docs/How-To-Write-New-Detectors.md]: Damn it, good catch, fixed.

**Pull request merged**